cli/src/lib/validators/validatePackageExports.js

```diff
@@ -31,21 +31,20 @@ module.exports.validatePackageExports = async (
     const baseDir = path.dirname(paths.package)
 
     let valid = true
-    const entrypointBasename = normalizeExtension(
-        path.basename(config.entryPoints.lib)
-    )
+    const entrypointBasename = normalizeExtension(config.entryPoints.lib)
+    const relativeEntrypoint = path.relative('src', entrypointBasename)
```

Comment:
Should use `paths.src` instead of hard-coding `'src'`

Reply:
```suggestion
    const relativeEntrypoint = path.relative(paths.src, entrypointBasename)
```

Reply:
Thanks! 2dda1d9]